content/en/docs/setup/platform-setup/alicloud/index.md

```diff
@@ -11,8 +11,6 @@ owner: istio/wg-environments-maintainers
 test: n/a
 ---
 
-This page was last updated August 8, 2018.
-
 {{< boilerplate untested-document >}}
```

Comment:
The boilerplate has this text: `This vendor-provided document has not been tested on the Istio 1.9 release and may contain bugs.`  If you feel that is accurate, you can leave it in, else you could remove it.

Reply:
Yes, we can leave it in and go ahead. Thanks for your reply.